pandas/tests/extension/test_boolean.py

```diff
@@ -28,7 +28,7 @@
 
 
 def make_data():
-    return [True, False] * 4 + [np.nan] + [True, False] * 44 + [np.nan] + [True, False]
+    return [True, False] * 4 + [pd.NA] + [True, False] * 44 + [pd.NA] + [True, False]
```

Comment:
why is this necessary?

Reply:
I made this change to track the identical changes in test_integer.py and test_floating.py.  The it does not matter whether the NA values in this case are pd.NA, np.nan, or a mixture.  But I did think it better to make the three files consistent.  I'm happy to revert this change if you prefer.

Reply:
Agreed this should be removed. It's ideal to minimize extraneous changes